### ISSUE TITLE
[Arcade Services] Powershell / Linux Scripts for accessing Maestro infos

### ISSUE THREAD

**Issue by JohnTortugo** [dotnet/arcade#3363]
While working on refactoring some `post-build` PS scripts - [see this folder](https://github.com/dotnet/arcade/tree/master/eng/common/post-build) I noticed that I had seen similar routines on other places ([for instance](https://github.com/dotnet/arcade-services/blob/master/src/Maestro/tests/Scenarios/common.ps1#L244)). Wouldn't it make sense to create a package with PS/"Bash" routines to interface with Darc / MaestroAPI? 

/cc @mmitche @riarenas @jcagme

**Comment by mmitche** [comment]
@JohnTortugo yeah, it would be useful.  Btw, that common file should use `darc` as much as possible (this is for scenario testing)
